src/idl_parser.cpp

```diff
@@ -3200,9 +3200,15 @@ CheckedError Parser::ParseRoot(const char *source, const char **include_paths,
                                const char *source_filename) {
   ECHECK(DoParse(source, include_paths, source_filename, nullptr));
 
+  auto leaks_private_implemtation = false;
+  auto is_private = false;
   // Check that all types were defined.
```

Comment:
@CasperN I'm not sure if this cpp code is the best in terms of quality, how would a person go about improving it?